### ISSUE TITLE
Latest version is failing to install inside node image

### ISSUE THREAD

**Issue by augustocdias** [Redocly/redoc#1641]
I'm getting this error when I try to install the latest version (the 0.11 works fine):

```
# npm install -g redoc-cli
npm ERR! code ECONNREFUSED
npm ERR! syscall connect
npm ERR! errno ECONNREFUSED
npm ERR! FetchError: request to http://0.0.0.0:4873/url-template/-/url-template-2.0.8.tgz failed, reason: connect ECONNREFUSED 0.0.0.0:4873
npm ERR!     at ClientRequest.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/minipass-fetch/lib/index.js:97:14)
npm ERR!     at ClientRequest.emit (node:events:394:28)
npm ERR!     at Socket.socketErrorListener (node:_http_client:447:9)
npm ERR!     at Socket.emit (node:events:406:35)
npm ERR!     at emitErrorNT (node:internal/streams/destroy:193:8)
npm ERR!     at emitErrorCloseNT (node:internal/streams/destroy:158:3)
npm ERR!     at processTicksAndRejections (node:internal/process/task_queues:83:21)
npm ERR!  FetchError: request to http://0.0.0.0:4873/url-template/-/url-template-2.0.8.tgz failed, reason: connect ECONNREFUSED 0.0.0.0:4873
npm ERR!     at ClientRequest.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/minipass-fetch/lib/index.js:97:14)
npm ERR!     at ClientRequest.emit (node:events:394:28)
npm ERR!     at Socket.socketErrorListener (node:_http_client:447:9)
npm ERR!     at Socket.emit (node:events:406:35)
npm ERR!     at emitErrorNT (node:internal/streams/destroy:193:8)
npm ERR!     at emitErrorCloseNT (node:internal/streams/destroy:158:3)
npm ERR!     at processTicksAndRejections (node:internal/process/task_queues:83:21) {
npm ERR!   code: 'ECONNREFUSED',
npm ERR!   errno: 'ECONNREFUSED',
npm ERR!   syscall: 'connect',
npm ERR!   address: '0.0.0.0',
npm ERR!   port: 4873,
npm ERR!   type: 'system'
npm ERR! }
npm ERR!
npm ERR! If you are behind a proxy, please make sure that the
npm ERR! 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-06-09T13_55_30_785Z-debug.log
```

**Comment by tkintscher** [comment]
I can sort of confirm, but the error message here refers to a different package:
```
❯ npm install redoc-cli
npm ERR! code ECONNREFUSED
npm ERR! syscall connect
npm ERR! errno ECONNREFUSED
npm ERR! FetchError: request to http://0.0.0.0:4873/slugify/-/slugify-1.4.7.tgz failed, reason: connect ECONNREFUSED 0.0.0.0:4873
...
```

**Comment by RomanHotsiy** [comment]
Fixed in 0.12.1.

Thanks for the report!
